index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="main-div">
+        <!--HEADER SECTION-->
+          <section class="header-section">
```

Comment:
- [ ] I suggest that you instead make this container of the section to have the `header` tag. This is because typically, when the `<header>` element is used as a page header, it contains the site’s title or logo (or both) and the main site navigation.